### PR TITLE
CD-172733 Add metadata_for

### PR DESCRIPTION
**Pull request by johnny-lai**
- [x] @johnwu96822 
- [x] @abdollar

### CODE REVIEW

File: lib/rdkafka/admin_operations.rb [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+        @native_kafka, 0, t_p,
+        metadata_p, timeout * 1000
+      )
+      raise Rdkafka::RdkafkaError.new(err) unless err == 0
+
+      metadata = Rdkafka::Metadata.from_native(metadata_p[:value])
+      return metadata.topics.first
+
+    ensure
+      if metadata_p && !metadata_p[:value].null?
```
Comment (johnwu96822) [inline lib/rdkafka/admin_operations.rb]: `nil?`

**Comment by abdollar** [comment]
👍

File: lib/rdkafka/admin_operations.rb (hunk 1 above)
Comment (johnny-lai) [reply lib/rdkafka/admin_operations.rb]: It's https://www.rubydoc.info/github/ffi/ffi/FFI/Pointer#null%3F-instance_method

**Comment by johnny-lai** [comment]
[lockit v4.9.2] merged via Lockit CLI on 2019-09-23 16:46:51.574012 -0700 PDT m=+6.654537866

**Pull request merged**